src/manifolds/MultinomialDoublyStochastic.jl

```diff
@@ -60,44 +98,27 @@ end
 Checks whether `p` is a valid point on the [`MultinomialDoubleStochastic`](@ref)`(n)` `M`,
 i.e. is a  matrix with positive entries whose rows and columns sum to one.
 """
-function check_point(
-    M::MultinomialDoubleStochastic,
-    p::T;
-    atol::Real=sqrt(prod(representation_size(M))) * eps(real(float(number_eltype(T)))),
-    kwargs...,
-) where {T}
+function check_point(M::MultinomialDoubleStochastic, p::T; kwargs...) where {T}
```

Comment:
```suggestion
function check_point(M::MultinomialDoubleStochastic, p; kwargs...)
```
We don't need that now.